deploy/olm-catalog/bundle/manifests/migration.openshift.io_miganalytics.yaml

```diff
@@ -62,14 +63,19 @@ spec:
           description: MigAnalyticSpec defines the desired state of MigAnalytic
           properties:
             analyzeImageCount:
+              description:  It is a boolean flag to enable analysis of image count. This is a required field.
               type: boolean
             analyzeK8SResources:
+              description: It is a boolean flag to enable analysis of k8s resources. This is a required field.
```

Comment:
I would remove `it is a` from all of these.

Reply:
Removed it.